webrecorder/replayweb.page
# Show snapshot timestamp in location bar

Resolves https://github.com/webrecorder/replayweb.page/issues/210

## Changes
- Shows how many snapshots are available per page in URL bar
- Allows users to navigate through snapshots through timestamp dropdown
- Refactors coll.js and pageutils to typescript

## Testing
1. Generate wacz with multiple timestamps by using the AWP and navigating to the same page at different times.
2. Load wacz and click page with multiple timestamps. Verify timestamps are shown in URL bar
3. Click timestamp in URL bar. Verify timestamp options are shown
4. Click option. Verify RWP loads with new timestamp
5. Click page with single snapshot Verify timestamp is still shown

## Screenshots
**Location bar:**
<img width="1504" alt="Screenshot 2023-08-30 at 12 52 56 PM" src="https://github.com/webrecorder/replayweb.page/assets/4672952/3e631702-2653-4742-b124-2290a440fcf0">

**Snapshot dropdown:**
<img width="283" alt="Screenshot 2023-08-30 at 12 53 00 PM" src="https://github.com/webrecorder/replayweb.page/assets/4672952/449176f9-cc3c-4a1b-a55e-334e4382fa7b">
